docs/shrc.md

```diff
@@ -79,7 +79,7 @@ A shared classes cache can be *persistent* or *non-persistent* according to the
 - persistent caches are written to memory-mapped files and remain in place, even after a system is rebooted.
 - non-persistent caches exist in shared memory and are automatically removed when the operating system is restarted.
 
-By default, a shared classes cache is persistent, except on the z/OS&reg; platform, where persistent caches are not supported.
+By default, a shared classes cache is persistent.
```

Comment:
"except on the z/OS patform" is still relevant, we didn't make persistent caches the default on z/OS although they are supported now.

https://eclipse.dev/openj9/docs/version0.40/#support-for-persistent-shared-classes-cache-added-on-zos